staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

```diff
@@ -138,10 +138,10 @@ func TestProxyHandler(t *testing.T) {
 			apiService: &apiregistration.APIService{
 				ObjectMeta: metav1.ObjectMeta{Name: "v1.foo"},
 				Spec: apiregistration.APIServiceSpec{
-					Service:               &apiregistration.ServiceReference{},
```

Comment:
drop this commit completely, this got merged upstream and the secure test is below

Reply:
Dropped